pkg/scheduler/metrics/metrics.go

```diff
@@ -345,7 +352,6 @@ func UnschedulablePods() metrics.GaugeMetric {
 // Reset resets metrics
 func Reset() {
 	SchedulingLatency.Reset()
-	DeprecatedSchedulingLatency.Reset()
```

Comment:
@brancz 

I was thinking, we should provide a `Reset()` API in component-base.  Instead of removing it when we deprecate a metric, because it will lose the possibility to turn on by `--show-hidden-metrics-for-version`.

Reply:
sgtm

Reply:
+1